include/ck/tensor_operation/gpu/device/conv_tensor_rearrange_op.hpp

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
```

Comment:
appears to be new file with copyright starting in the past.

Reply:
Changed.